app/templates/components/ui-table/cell/events/view/speakers/cell-simple-sessions.hbs

```diff
@@ -1,8 +1,28 @@
-<div class="ui unordered list">
+<div class="ui list">
   {{#each record.sessions as |session|}}
     {{#if (not eq session.deletedAt)}}
       <div class="item">
-        {{session.title}}
+        {{#if (eq session.state "accepted")}}
```

Comment:
This code can be optimized a lot, return the color as a computed property based on the status of the session.